src/transformers/models/distilbert/modeling_distilbert.py

```diff
@@ -142,10 +141,7 @@ def __init__(self, config: PretrainedConfig):
         self.dim = config.dim
         self.dropout = nn.Dropout(p=config.attention_dropout)
 
-        # Have an even number of multi heads that divide the dimensions
-        if self.dim % self.n_heads != 0:
-            # Raise value errors for even multi-head attention nodes
-            raise ValueError(f"self.n_heads: {self.n_heads} must divide self.dim: {self.dim} evenly")
+        assert self.dim % self.n_heads == 0
```

Comment:
Could you also revert the changes here ? It seems that you have deleted this by mistake